cognite/client/_api/data_modeling/_data_modeling_executor.py

```diff
@@ -6,7 +6,7 @@
 
 _THREAD_POOL_EXECUTOR_SINGLETON: ThreadPoolExecutor
 _MAIN_THREAD_EXECUTOR_SINGLETON = MainThreadExecutor()
-_MAX_WORKERS = 2
+_MAX_WORKERS = 1
```

Comment:
Would it require more advanced logic to handle concurrent calls?

Reply:
No we just don't want a single user to use more than 1 token from the project's concurrency budget by default.

Reply:
Nice usage of `ThreadPoolExecutor` :trollface: